src/app/base/components/ZoneSelect/ZoneSelect.test.tsx

```diff
@@ -1,18 +1,18 @@
-import { mount } from "enzyme";
 import { Formik } from "formik";
-import { Provider } from "react-redux";
 import configureStore from "redux-mock-store";
 
 import ZoneSelect from "./ZoneSelect";
 
+import type { RootState } from "app/store/root/types";
 import {
   rootState as rootStateFactory,
   zone as zoneFactory,
   zoneGenericActions as zoneGenericActionsFactory,
   zoneState as zoneStateFactory,
 } from "testing/factories";
+import { renderWithMockStore, screen } from "testing/utils";
 
-const mockStore = configureStore();
+const mockStore = configureStore<RootState>();
```

Comment:
We're not directly interacting with the store in any of the test so you can remove this and pass the state to `renderWithMockStore` directly.

Reply:
How about line 52, where I have to test the store actions?